src/libraries/testPackages/packageSettings/System.Runtime.Caching/netstandard/settings.targets

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup>
+    <!-- intentional dangling ref in System.Configuration.ConfigurationManager -->
+    <IgnoredReference Include="System.Security.Permissions" />
```

Comment:
Interesting -- it looks like System.Runtime.Caching is the only library that retained it's control knob to from config file: https://github.com/dotnet/runtime/blob/0a15c3b97dabaf94e5d4399aef2fd48c2b3f446a/src/libraries/System.Runtime.Caching/src/System/Runtime/Caching/MemoryCacheStatistics.cs#L129

Probably because it was viewed as a "legacy" package with a replacement in Microsoft.Extensions -- similar to System.Configuration.ConfiguratinManager itself.